doc/fundamentals/append_overwrite.rst

```diff
@@ -1,571 +1,364 @@
-   above, if the flags supplied to ``put file`` are
-   ``--split lines --target-file-datums 2 --target-file-bytes 100``, it
-   will have the same result as ``--target-file-datums 2``, since that’s
-   the most compact constraint, and file sizes will hover around 40
-   bytes.
-
-What split data looks like in a Pachyderm repository
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Going back to our 50-line file example, let’s say that file is named
-“my-data.txt”. We’ll create a repo named “line-data” and load
-my-data.txt into Pachyderm with the following commands:
-
-::
-   
-   $ pachctl create repo line-data
-   $ pachctl put file line-data@master -f my-data.txt --split line
+line-delimited text files, JavaScript Object Notation (JSON) files,
+Pachyderm includes the `--split`, `--target-file-bytes`, and
+`--target-file-datums` flags.
 
-After ``put file`` is complete, list the files in the repo.
+**Note:** In this section, a chunk of data is called a *split-file*.
 
-::
-   
-   $ pachctl list file line-data@master
-   COMMIT                           NAME         TYPE COMMITTED          SIZE
-   8cce4de3571f46459cbe4d7fe222a466 /my-data.txt dir  About a minute ago 1.071KiB
+| Flag              | Description                                           |
+| ----------------- | ----------------------------------------------------- |
+| `--split`         | Divides a file into chunks based on a *record*. In <br>line-delimited files, it is a line. In JSON files, <br>it is an object. The `--split` flag takes one argument, such as `line`, <br> `json`, or `sql`. This argument defines how to split the file into <br>chunks. For example, if you use `--split line`, Pachyderm divides <br>your file on new line boundaries and not in the middle of a line. |
+| `--target-file-bytes` |  This flag must be used with the `--split` <br> flag. The `--target-file-bytes` fills each of the split-files with data up to <br> the specified number of bytes, splitting on the nearest <br>record boundary. For example, you have a line-delimited file <br>of 50 lines, with each line having about 20 bytes. If you run the <br> `--split lines --target-file-bytes 100` command, you see the input <br>file split into about 10 files and each file has about 5 lines. Each <br>split-file’s size hovers above the target value of 100 bytes, <br>not going below 100 bytes until the last split-file, which might <br>be less than 100 bytes. |
+| `--target-file-datums` | This flag must be used with the `--split` <br> flag. The `--target-file-datums` attempts to fill each split-file <br>with the specified number of datums. If you run `--split lines --target-file-datums 2` on the line-delimited 50-line file mentioned above, you see the file split into 50 split-files and each file has 2 lines. |
+
+
+If you specify both `--target-file-datums` and `--target-file-bytes` flags,
+Pachyderm creates split-files until it hits one of the
+constraints. Pachyderm splits the file and
```

Comment:
I think you can remove:

"Pachyderm splits the file and
then fills the first target split-file with line-based records
until it hits the record limit. If Pachyderm passes the target byte
number with just one record, it moves to the next split-file. If Pachyderm
hits the target datum number after adding another line, it moves to the
next split-file. If you run `pachctl put file` with
`--split lines --target-file-datums 2 --target-file-bytes 100` flags
on the example file in the table above, you get the same result as
`--target-file-datums 2`, since that is the most compact constraint,
and file sizes hover around 40 bytes."

everything there is already explained and it shortens things nicely.